2010-09-01  Miloslav Trmaƒç  <mitr@redhat.com>

	* libuser.conf (crypt_style): Change default to sha512.

diff -r f359c24398cc -r 276fa0a3078b libuser.conf
--- a/libuser.conf	Sun Aug 29 12:08:06 2010 +0000
+++ b/libuser.conf	Wed Sep 01 16:13:31 2010 +0200
@@ -17,7 +17,7 @@
 # skeleton = /etc/skel
 # mailspooldir = /var/mail
 
-crypt_style = md5
+crypt_style = sha512
 modules = files shadow
 create_modules = files shadow
 # modules = files shadow ldap
